packages/back-end/src/models/MetricModel.ts

```diff
@@ -550,3 +551,33 @@ export async function removeTagInMetrics(organization: string, tag: string) {
     }
   );
 }
+
+export async function generateMetricEmbeddings(
+  context: ReqContext | ApiReqContext,
+  metricsToGenerateEmbeddings: MetricInterface[]
```

Comment:
This is the legacy metric interface. We should really only be supporting fact metrics going forward with new features like this.